sdk/core/core-http/src/serviceClient.ts

```diff
@@ -555,16 +557,18 @@ export function serializeRequestBody(
         const isStream = typeName === MapperType.Stream;
 
         if (operationSpec.isXML) {
+            const xmlnsKey = xmlNamespacePrefix ? `xmlns:${xmlNamespacePrefix}` : "xmlns";
+            const value = xmlNamespace && !["Composite", "Sequence"].includes(typeName) ? {_: httpRequest.body, $: {[xmlnsKey]: xmlNamespace}} : httpRequest.body;
```

Comment:
this one liner is a bit too complicated for me - can we break it into an if/else?